cmd/container-use/main_suite_test.go

```diff
@@ -20,15 +21,19 @@ var (
 func getContainerUseBinary(t *testing.T) string {
 	binaryPathOnce.Do(func() {
 		t.Log("Building fresh container-use binary...")
-		cmd := exec.Command("go", "build", "-o", "container-use", ".")
+		binaryName := "container-use"
+		if runtime.GOOS == "windows" {
+			binaryName = "container-use.exe"
+		}
```

Comment:
style nit, but kinda a big one: use build directives instead of conditional logic -- maybe fix in a follow-up?